modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsForm.es.js

```diff
@@ -492,18 +461,20 @@ class ResultRankingsForm extends Component {
 					totalResultsHiddenCount: total
 				}));
 			})
-			.catch(() => {
+			.catch(error => {
 				// Delay showing error message so the user has confirmation
 				// when attempting to reload the content after an error.
 
-				setTimeout(
-					() =>
-						this.setState({
-							dataLoadingHidden: false,
-							displayErrorHidden: true
-						}),
-					1000
-				);
+				setTimeout(() => {
+					if (process.env.NODE_ENV === 'development') {
+						console.error('Unable to load content due to ', error);
```

Comment:
These will print on separate lines rather than concatenation. `console.error('Unable to load content.', error);` could be more appropriate.